Lecture2/Task1/builder/GuitarBuilder.java

```diff
@@ -0,0 +1,27 @@
+package builder;
+
+import product.Brand;
+
+public class GuitarBuilder {
```

Comment:
what is the responsibility of this class? 
you have all this method in Guitar class - use it instead

Reply:
discussed